community/mei-by-laws.md

```diff
@@ -12,7 +12,7 @@ The name of this organisation is ‘Music Encoding Initiative’ (hereafter abbr
 
 The purpose of MEI is to create, promote, and diffuse a semantically rich model for music notation that:
 
-* accommodates the encoding of the intellectual and physical characteristics of Western music notation documents, primarily focusing on but not being limited to Common Music Notation (CMN);
+* accommodates the encoding of the intellectual and physical characteristics of music notation documents, primarily focusing on but not being limited to Common Western Music Notation (CWMN);
```

Comment:
punctuation: "...primarily focusing on, but not being limited to, Common Western..."

Reply:
Thank you